### ISSUE TITLE
why not stop gradient for the target_flow_actions in distill loss

### ISSUE THREAD

**Issue by Triple-ROCK** [seohongpark/fql#8]
https://github.com/seohongpark/fql/blob/e8cd16eb490332924dfa2492097219f181765933/agents/fql.py#L64

Hello, Thanks for you great work. While I am reproducing your work, I found a conflicts between the idea in your paper and the implementation. The paper said "Naively doing this with a flow policy involves costly and unstable backpropagation through time (BPTT)". However, if you don't do jax.lax.stop_gradient(target_flow_actions), then you are essentially doing BPTT.

**Comment by seohongpark** [comment]
Hi Triple-ROCK,

In our code, (informally) everything is "stop-gradiented" by default (when calling a network), and we need to explicitly pass `grad_params` to flow gradients. So L64 doesn't flow gradients to the flow vector field (cf. L65, where we explicitly state `params=grad_params`).
